cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -33,8 +35,21 @@ public boolean install(boolean silent) {
   @Override
   public void installPlugin(PluginDescriptor plugin) {
 
-    // TODO Auto-generated method stub
-
+    // no override needed
   }
 
+  @Override
+  public void postInstall() {
+
+    super.postInstall();
+
+    Path binPath = getToolPath().resolve("bin");
+    if (this.context.getSystemInfo().isLinux()) {
```

Comment:
use if `(SystemInfoImpl.INSTANCE.isLinux())`. the other one is inconsistent I heard.

Reply:
the method is already deleted